### PR TITLE
FIX: The cjs template does not employ the ts template's db and server connection method/approach yet 

### LINKED ISSUE #13: The cjs template does not employ the ts template's db and server connection method/approach yet
#### Issue description
The cjs template does not employ the ts template's db and server connection method/approach yet. The ts approach is straightforward and better structured.

#### Steps to reproduce the issue
1. Go to https://github.com/code-collabo/node-mongo-api-boilerplate-templates
2. Compare cjs template's db and server files to that of the ts template
3. You will see that the ts template uses a more structured approach in the way the db and server is connected

#### Expected behavior
The cjs template should use the mongooseConnect() and mongooseAtlasConnect() functions where necessary, and adopt the file names, in order to look consistent with the ts template. 

#### Screenshots
None

#### Additional context
None

#### Testing checklist
Note: Add the testing checklist below (without this note) to your pull request when submitting your fix. This will guide (you and) the contributors who will review your fix to know what is important to check or test for.
- [ ] mongooseConnect() function is now added and used in the cjs template, just as it is used in the ts template
- [ ] mongooseAtlasConnect() function is now added and used in the cjs template, just as it is used in the ts template
- [ ] all db and server files  in the cjs template now adopt the file names as used in the ts template
- [ ] No new code or functionality was added, I only made the cjs template to employ the ts template's db and server connection method/approach
- [ ] I certify that I ran my checklist

### PR DESCRIPTION
**Pull request by samuko-things**
**This pull request makes the following changes**
* Fixes code-collabo/node-mongo#13

**General checklist**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**
- [x] mongooseConnect() function is now added and used in the cjs template, just as it is used in the ts template
- [x] mongooseAtlasConnect() function is now added and used in the cjs template, just as it is used in the ts template
- [x] all db and server files in the cjs template now adopt the file names as used in the ts template
- [x] No new code or functionality was added, I only made the cjs template to employ the ts template's db and server connection method/approach
- [x] I certify that I ran my checklist

Ping @code-collabo/node-mongo

### CODE REVIEW

**Comment by samuko-things** [comment]
@Ifycode, i have just updated the cjs template db connection to match that of ts template, do check it out. I am only concerned about editing the package.json file.

**Comment by Ifycode** [comment]
@samuko-things check my review comment here. You will have a better idea: https://github.com/code-collabo/node-mongo-api-boilerplate-templates/pull/20#discussion_r1174687736

**Comment by Ifycode** [comment]
@samuko-things I've added a more comprehensive comment for the package.json issue/part. Please take a look: https://github.com/code-collabo/node-mongo-api-boilerplate-templates/pull/20#discussion_r1174708318

**Comment by samuko-things** [comment]
i have updated the package.json file as suggested as well as some other files. please do check it out. i have also tested it and it works.

File: cjs/src/server.js [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
-
-server.listen(port, () => {
-  try {
-    console.log( chalk.cyanBright(`\nServer running at http://localhost:${port}`) );
-  }catch(err) {
-    console.log(err);
-  }
+app.listen(port, () => {
+  mongooseConnect(port);
 });
```
Comment (Ifycode) [inline cjs/src/server.js:8]: Thank you @samuko-things 
Approving and sending over to a cleanup branch for further cleanup and testing

**Review by Ifycode — APPROVED** [review]

**Pull request merged**